bin/compile

```diff
@@ -29,4 +29,13 @@ if ! [ $? ]; then
     exit 1
 fi
 
+if [ -f "${BUILD_DIR}/${APP_SUBFOLDER}/yarn.lock" ]; then
```

Comment:
seems hacky but ok

Reply:
its as hacky as the rest of the build pack 🤷‍♂️